blog/20140130-mate-desktop-live-cd.de.md

```diff
@@ -70,22 +70,22 @@ aber wir haben einige zusätzliche nicht zum MATE-Desktop gehörenden Anwendunge
 um die Live-CD ein wenig nützlicher und unterhaltsamer zu machen.
 
   * [Firefox](https://www.mozilla.org/firefox/)   - Stand-Alone Web-Browser von mozilla.org
-  * [GParted](http://gparted.sourceforge.net)   - Ein Klon von Partition Magic, Front-End für GNU Parted
-  * [HardInfo](http://hardinfo.berlios.de/wiki/index.php/Main_Page)  - Ein System-Informations- und Benchmark-Tool
-  * [Hexchat](http://hexchat.github.io/)   - IRC-Client, konfiguriert für die automatischen Verbindung zu [#mate@freenode](https://webchat.freenode.net/?channels=#mate)
-  * [Onboard](http://www.gnome.org/projects/orca)      - Screen-Reader für blinde bzw. sehbeinträchtigte Personen
-  * [Orca](http://www.gnome.org/projects/orca)      - On-Screen-Keyboard, nützlich für motorisch eingeschränkte Personen
-  * [Pidgin](http://pidgin.im/)    - Multi-Protokoll Instant-Messaging Client
-  * [Truecrypt](http://www.truecrypt.org/) - Freie, quelloffene, cross-plattform Festplatten-Verschlüsselungs-Software
-  * [Xnoise](http://www.xnoise-media-player.com/)    - Medien-Player mit einer schicken GUI, hoher Geschwindigkeit und vielen Features
+  * [GParted](https://gparted.sourceforge.net)   - Ein Klon von Partition Magic, Front-End für GNU Parted
+  * [HardInfo](https://hardinfo.berlios.de/wiki/index.php/Main_Page)  - Ein System-Informations- und Benchmark-Tool
```

Comment:
https://hardinfo.berlios.de/wiki/index.php/Main_Page seems to redirect to wrong page

maybe https://www.berlios.de/software/hardinfo/ is better?

Reply:
Will change it.